src/components/box/box.stories.mdx

```diff
@@ -279,5 +365,17 @@ If no `scrollVariant` prop is set then the scroll bar will resort to the browser
       description: "String to denote which style variant to use for scrollbars",
       required: false,
     },
+    {
+      name: "boxSizing",
+      type: { summary: "content-box | border-box" },
+      description: "Set the box-sizing attribute of the Box component",
+      required: false,
+    },
+    {
+      name: "tabIndex",
```

Comment:
Good spot. I totally forgot to add this when I added tabIndex 🙄

Reply:
all good there was a few missing